cmd/utils/flags.go

```diff
@@ -1161,7 +1161,7 @@ func setSmartCard(ctx *cli.Context, cfg *node.Config) {
 	// Sanity check that the smartcard path is valid
 	fi, err := os.Stat(path)
 	if err != nil {
-		log.Error("Failed to verify smartcard daemon path", "path", path, "err", err)
+		log.Info("smartcard daemon scoket path not found, disabling", "path", path, "err", err)
```

Comment:
Please capitalize and make it shorter than 40 characters :)

Reply:
"SmartCard daemon file missing, disabling scwallet"

Reply:
You can omit the "path" field because the path will always be in the error message.